Add Copilot plugin to MCP setup instructions for VS Code (#37949)
DataDog/documentation: content/en/mcp_server/setup.md

{{% tab "VS Code" %}}

> For Copilot, you should install the [Datadog Copilot plugin][2] from the marketplace (follow the same instructions as for the [Copilot CLI][3]).

Reviewer: ```suggestion
For Copilot, install the [Datadog Copilot plugin][2] from the marketplace. For more information, see the instructions for the [Copilot CLI][3].
```

1. Run the {{< ui >}}Datadog: Open MCP Configuration Assistant{{< /ui >}} and follow the guidance to configure the Datadog MCP Server.
1. Verify that you have the required [permissions](#required-permissions) for the Datadog resources you want to access.

> The Datadog MCP server connection is managed by Copilot (or whichever agent you are using) not the Datadog extension. You will be required to authorize the Datadog MCP server independently of the extension.

Reviewer: ```suggestion
The Datadog MCP server connection is managed by Copilot (or whichever agent you are using), not the Datadog extension. You must authorize the Datadog MCP server independently of the extension.
```
